demos/ros_examples/README.md

```diff
@@ -0,0 +1,34 @@
+# `ros_examples` demonstration scripts
+
+## Table of contents:
+* [Running a demo script](#running-demonstration-scripts)
+* [Task Space control loop example](#task_space_control_loop)
+
+## Running demonstration scripts
+This package contains the same control loop examples as in [`control_loop_examples`](../control_loop_examples) demos
+folder, but adapted for usage in a ROS environment.
+The whole folder is a fully functional ROS package that can be directly copied in a ROS workspace.
+There is also a provided `Dockerfile` that encapsulates the whole in a containerized ROS environment.
+Running the `run-demo.sh` script opens an interactive container with the ROS environment already setup for running
+the demonstration scripts.
+
+In order to run the demonstrations, a `roscore` needs to be running.
```

Comment:
Does this need two containers running, one with roscore and the other one rosrun or can the roscore be on the host machine directly? I think its the latter, but we should say that explicitly.

Reply:
it can be both. You can have the roscore on the host or in a second container